src/js/components/Tab/Tab.js

```diff
@@ -75,6 +76,9 @@ const Tab = forwardRef(
         event.preventDefault();
       }
       onActivate();
+      if (onClick) {
```

Comment:
The onMouse ones don't care. Any hint what type of situation would cause that? I'm not sure why you wouldn't want it to since it would have been called if we didn't have an internal handler

Reply:
not sure, but noticed the check on line 74 above.